Makefile

```diff
@@ -64,7 +64,7 @@ update-operator-crds:
 check-operator-crds:
 	APP_VERSION=$$(cat ./charts/opentelemetry-operator/Chart.yaml | sed -nr 's/appVersion: ([0-9]+\.[0-9]+\.[0-9]+)/\1/p'); \
 	mkdir -p ${TMP_DIRECTORY}/crds; \
-	curl -s -o "${TMP_DIRECTORY}/crds/crd-opentelemetrycollector.yaml" https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/v$${APP_VERSION}/config/crd/bases/opentelemetry.io_opentelemetrycollectors.yaml; \
+	#curl -s -o "${TMP_DIRECTORY}/crds/crd-opentelemetrycollector.yaml" https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/v$${APP_VERSION}/config/crd/bases/opentelemetry.io_opentelemetrycollectors.yaml;
```

Comment:
I was thinking about suing the CRD from the bundle https://github.com/open-telemetry/opentelemetry-operator/blob/main/bundle/manifests/opentelemetry.io_opentelemetrycollectors.yaml 

but the webhook service differs in the helm chart - it does not have the `-service` suffix.